src/integration/tests/firstTest.test.tsx

```diff
@@ -13,39 +12,114 @@ import {
   INPUT_TITLE_ITEM_ID,
   INPUT_VALUE_ITEM_ID,
   BUTTON_ADD_ITEM_ID,
-  NEXT_MONTH_ID,
-  PREV_MONTH_ID,
+  BUTTON_NEXT_MONTH_ID,
+  BUTTON_PREV_MONTH_ID,
   CURRENT_MONTH_ID,
+  BALANCE_VALUE_ITEM_ID,
+  REVENUE_VALUE_ITEM_ID,
+  EXPENSE_VALUE_ITEM_ID,
 } from "../helpers/testHelper";
 
-import { HEADER_TEXT,
-  INFO_TEXT_RECEITAS,
-  INFO_TEXT_DESPESAS,
-  INFO_TEXT_BALANCO,
-  INPUT_TEXT_DATA,
-  INPUT_TEXT_CATEGORIA,
-  INPUT_TEXT_TITULO,
-  INPUT_TEXT_VALOR,
-  INPUT_TEXT_ADICIONAR,
-  TABLE_TEXT_DATA,
-  TABLE_TEXT_CATEGORIA,
-  TABLE_TEXT_TITULO,
-  TABLE_TEXT_VALOR } from "../strings/testStrings";
+import {
+  SYSFINAN_LABEL,
+  REVENUE_LABEL,
+  EXPENSE_LABEL,
+  BALANCO_LABEL,
+  DATE_LABEL,
+  CATEGORY_LABEL,
+  TITLE_LABEL,
+  VALUE_LABEL,
+  ADD_LABEL,
+} from "../strings/testStrings";
 
 describe("First test", () => {
-/*   beforeEach(() => {
+  beforeEach(() => {
     render(<App />);
-  }); */
+  });
+
+  it("should be possible to view the title Sistema Financeiro.", () => {
+    const tableArea = within(screen.getByTestId(HEADER_AREA_ID));
+    const headerText = tableArea.getByText(SYSFINAN_LABEL);
+    expect(headerText).toBeInTheDocument();
+  });
+
+  it("should be possible to view the fields Receitas, Despesas, and Balanço in the information area", () => {
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+
+    const infoReceita = infoArea.getByText(REVENUE_LABEL);
+    const infoDespesas = infoArea.getByText(EXPENSE_LABEL);
+    const inforBalanco = infoArea.getByText(BALANCO_LABEL);
+
+    expect(infoReceita).toBeInTheDocument();
+    expect(infoDespesas).toBeInTheDocument();
+    expect(inforBalanco).toBeInTheDocument();
+  });
+
+  it("should be possible to view the date field in the format Month de Year in the information area.", () => {
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+    function formatDate(date = new Date()) {
+      const options: Intl.DateTimeFormatOptions = {
+        month: "long",
+        year: "numeric",
+      };
+      const formattedDate = date.toLocaleString("pt-BR", options);
+      const [month, year] = formattedDate.split(" de ");
+      return `${month.charAt(0).toUpperCase()}${month.slice(1)} de ${year}`;
+    }
+
+    const dateFormat = infoArea.getByText(formatDate());
```

Comment:
This is not wrong, but to simplify I think the "moment" should work here to format the date. Could you try it?

https://momentjs.com/